src/entities/tag.cpp

```diff
@@ -469,7 +482,8 @@ bool Tag::store() {
     if (this->id > 0) {
         query.prepare(
                 "UPDATE tag SET name = :name, priority = :priority, "
-                        "parent_id = :parentId, " + colorField + " = :color "
+                        "parent_id = :parentId, " + colorField + " = :color, "
+                        "created = datetime('now') "
```

Comment:
> One can also argue, that if you rename a tag for instance, you basically create a new one with the same id while deleting the old one

where did I do that?
here is where renaming is done: <https://github.com/pbek/QOwnNotes/blob/develop/src/mainwindow.cpp#L6522>

> I would rather vote to rename created to updated.
The problem is that there is no easy way to rename columns in SQLite...